client/flutter/lib/l10n/your_questions_answered.yaml

```diff
@@ -0,0 +1,35 @@
+schema_version: 1
+contents:
+  type: qa
+  items:
+    -
+    title: |
```

Comment:
Is this html or text?

Reply:
both the title and body are html.  I will change the keys to title_html and body_html.   Sound right?